cypress/automated-tests/youth-pass/address-section-required-fields.spec.js

```diff
@@ -4,8 +4,14 @@ describe('Youth Pass Address Section Required Fields', () => {
     it('proceeds through an application', () => {
         const applicantZipCode = '02114';
         const youthPassUrl = Cypress.env('youth_pass_url');
-        const randomBirthdate = faker.date.between('2003-11-02', '2009-11-01');
-        const applicantBirthdate = `${randomBirthdate.getMonth() + 1}/${randomBirthdate.getDate()}/${randomBirthdate.getFullYear()}`;
+        const todaysDate = new Date();
+        const twelveYearsAgo = todaysDate.getFullYear() - 12; 
+        const eighteenYearsAgo = todaysDate.getFullYear() - 18; 
+        const randomBirthdate = faker.date.between(`${eighteenYearsAgo}-11-02`, `${twelveYearsAgo}-11-01`);
+        const applicantBirthdate = 
+            `${randomBirthdate.getMonth() + 1}/
+            ${randomBirthdate.getDate()}/
+            ${randomBirthdate.getFullYear()}`.replace(/\s+/g, "");    
```

Comment:
```suggestion
        const applicantBirthdate =  `${randomBirthdate.getMonth() + 1}/${randomBirthdate.getDate()}/${randomBirthdate.getFullYear()}`;
```

suggestion: I suspect the spacing you are seeing in the values is due to the line breaks here, but if you keep your string literal on a single line I think you might not need to go through the replacement step.

Reply:
Got it. I was trying to respect line length here, but it's much simpler to keep these literals on a single line